data/scenarios/tutorial.sav

```diff
@@ -334,7 +337,7 @@ function tutorial_unit_cant_be_built_callback(unittype, city, reason)
   end
   if unittype.id == find.unit_type('Settlers').id and not nosettlermsg then
     notify.event(city.owner, city.tile, E.SCRIPT,
-_("No Settler at size 2 city: Your city cannot build a Settler at this time. Settlers take two units of population to build, so a city of size one or two cannot build one without disbanding the city. To remedy this, you need to adjust the citizens in the city to provide more food so as to grow the city faster. Cities that do not have much food should not try to build settlers right away. When founding a new city, make sure it is built on terrain that provides enough food -- grassland is best; plains or hills are almost as good expecially if they have resources on them such as buffalo or coal."))
+_("No Settler at size 2 city: Your city cannot build a Settler at this time. Settlers take two units of population to build, so a city of size one or two cannot build one without disbanding the city. To remedy this, you need to adjust the citizens in the city to provide more food so as to grow the city faster. Cities that do not have much food should not try to build Settlers right away. When founding a new city, make sure it is built on terrain that provides enough food -- grassland is best; plains or hills are almost as good expecially if they have resources on them such as buffalo or coal."))
```

Comment:
Can we use an em dash instead of a double hyphen?

Typo: "especially", not "expecially"